specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerEffectiveConfiguration.json

```diff
@@ -89,7 +89,7 @@
   "definitions": {
     "EffectiveConfiguration": {
       "properties": {
-        "configType": {
+        "commitType": {
```

Comment:
with such a change - how will your UI/clients handle this?

Reply:
We will discuss and make a plan